2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,16 @@
   
 */
 
-function getSettlers() {}
+
+function getSettlers(familiesArray) {
+  let shouldStay = [];
+  for( let i = 0; i < familiesArray.length; i++){
+    if(familiesArray[i].startsWith("A") && familiesArray[i].includes("family")){
+        shouldStay.push(familiesArray[i]);
+    }
+  }
```

Comment:
this works well and your variable name of `shouldStay` really helps with readability. You could also use the `.filter()` method here